### DISCUSSION TITLE
New Here

### DISCUSSION THREAD

**Discussion by sxqib** [community/community#36256] [GitHub Education]
Hello Guys! I am a student who is interested in Python development! nice to see you!

**Comment by radar-blip** [comment]
Hi @saq1bx , welcome aboard!!

**Comment by ossanna16** [comment]
👋 Welcome to the GitHub Community on Discussions, @saq1bx! Happy to have you here 🙂
To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204). To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).

**Reply by sxqib** [reply]
thank you!

**Comment by PARTHIB-DEB** [comment]
Hello Guys 😄 
I am a backend web developer . I do backend in django and prefer postgres as database. Although I have done some certifications on Mongodb to connect in python also. How I can be a web developer if I don't know frontend ?? 🤔 So yeah! I do frontend in Vanilla HTML,CSS,JS as well as with tailwind. That's a little synopsis !! Let's connect guys 👋
